.github/workflows/plugin_install.yml

```diff
@@ -3,63 +3,121 @@ name: Plugin Install
+          chmod +x plugins/opensearch-security/tools/install_demo_configuration.sh
+          yes | plugins/opensearch-security/tools/install_demo_configuration.sh
+          chown 1001:1001 -R /opensearch
+          su -c "/opensearch/bin/opensearch" -s /bin/bash opensearch
+          EOF
+          docker build -t opensearch-test:latest -f- . <<EOF
+          FROM ubuntu:latest
+          COPY --chown=1001:1001 os-ep.sh /docker-host/
+          COPY --chown=1001:1001 opensearch-security.zip /docker-host/security-plugin.zip
+          COPY --chown=1001:1001 opensearch* /opensearch/
+          RUN chmod +x /docker-host/os-ep.sh
+          RUN useradd -u 1001 -s /sbin/nologin opensearch
+          ENV PATH="/opensearch/bin:${PATH}"
+          WORKDIR /opensearch/
+          ENTRYPOINT /docker-host/os-ep.sh
+          EOF
+          docker run --name ops -d -p 9200:9200 -p 9600:9600 -i opensearch-test:latest
+
+      - name: Sleep while OpenSearch finishes starting up
+        uses: whatnick/wait-action@v0.1.2
+        with:
+          time: '30s'
+
+      - name: Check OpenSearch Running
+        run: curl -XGET https://localhost:9200 -u 'admin:admin' -k -v
+
+      - name: Get Docker Logs
+        if: always()
+        run: docker logs ops
+
+      - name: Run sanity tests
+        run: ./gradlew integTestRemote -Dtests.rest.cluster=localhost:9200 -Dtests.cluster=localhost:9200 -Dtests.clustername="opensearch" -Dhttps=true -Duser=admin -Dpassword=admin
+
+  windows-install:
+    if: ${{ false }}  # disable for now - no stable distribution for 3.0.0 yet
+
+    runs-on: windows-latest
+
+    steps:
+      - name: Enable longer filenames for Windows
+        run: git config --system core.longpaths true
+
+      - name: Checkout Plugin
+        uses: actions/checkout@v2
+
+      - name: Set up JDK
+        uses: actions/setup-java@v1
+        with:
+          java-version: 11
+
+      - name: Download OpenSearch Core
+        run: |
+          cd ..
+          Invoke-WebRequest https://artifacts.opensearch.org/snapshots/core/opensearch/3.0.0-SNAPSHOT/opensearch-min-3.0.0-SNAPSHOT-windows-x64-latest.zip -Outfile opensearch-3.0.0.zip
+          tar -xzf opensearch-3.0.0.zip 
+          del opensearch-3.0.0.zip
+          Rename-Item opensearch-* Opensearch
+
+      - name: Build Security Plugin with Gradle
+        run: |
+          .\gradlew.bat assemble
+        env:
+          _JAVA_OPTIONS: -Xmx4096M
+
+      - name: Move and rename security plugin
+        run: move build\distributions\opensearch-security-*.zip ..\opensearch-security.zip
+
+      - name: Install the plugin
+        run: |
+          cd ..
+          'y' | .\Opensearch\bin\opensearch-plugin.bat install file:\a\security\opensearch-security.zip
+          'y', 'y', 'N' | .\Opensearch\plugins\opensearch-security\tools\install_demo_configuration.bat
+
+      - name: Run OpenSearch with plugin
+        run: |
+          cd ..
+          start .\Opensearch\bin\opensearch.bat
```

Comment:
We should have a way to get the logs from the process in cases it doesn't come up like in the linux version

https://github.com/opensearch-project/security/blob/main/.github/workflows/plugin_install.yml#L60-L62

Reply:
Note; this is a good candidate for a follow up issue